projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -169,28 +402,46 @@ export function navigateToQuoteListFromQuoteDetails() {
 
 /**
  * Checks the displayed quote, assuming that it is in draft state.
- * @param meetsThreshold does the quote meet the threshold
- * @param productId product id of a product which is part of the quote
+ * @param meetsThreshold Does the quote meet the threshold
+ * @param productId Product id of a product which is part of the quote
  */
 export function checkQuoteInDraftState(
   meetsThreshold: boolean,
   productId: string
 ) {
-  checkQuoteState('Draft');
-  this.checkGlobalMessageDisplayed(!meetsThreshold);
-  this.checkSubmitButton(meetsThreshold);
-  cy.get('.cx-code').should('contain.text', productId);
+  log(checkQuoteInDraftState.name);
+  checkQuoteState(STATUS_DRAFT);
+  checkGlobalMessageDisplayed(!meetsThreshold);
+  checkSubmitButton(meetsThreshold);
+  checkItemInQuoteCart(productId);
+}
+
+/**
+ * Checks if the given item exists within the quote cart
+ * @param productId Product ID of the item that should exist in the cart
+ */
+export function checkItemInQuoteCart(productId: string) {
+  log(checkItemInQuoteCart.name);
+  cy.get('cx-quote-details-cart .cx-table-item-container .cx-info').contains(
+    productId
+  );
 }
 
+/**
+ * Checks if the quote has the given status
+ * @param status Expected Status of the quote
+ */
 export function checkQuoteState(status: string) {
+  log(checkQuoteState.name);
```

Comment:
I would suggest not to log the name of the method but to log the method description, for instance:
> Verifies the quote state

Reply:
done